app/knights_person/knight.py

```diff
@@ -0,0 +1,22 @@
+class Knight:
+    def __init__(self, name: str, hp: int, power: int) -> None:
+        self.name = name
+        self.hp = hp
+        self.protection = 0
+        self.power = power
+
+    def __str__(self) -> str:
+        return f"name: {self.name}, hp: {self.hp}, " \
+               f"protection: {self.protection}, power: {self.power}"
+
+    def set_protection(self, armours: list) -> None:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]